Update and warn the users on how to remove detailed geometries (#131)
QGEP/docs: en/user-guide/digitizing/deletingobjects.rst

* Click the red x button to delete the structure parts
* Click save on the form
* Stop the editing mode and confirm changes to layer. All changes will be saved to database.

> Delete detailled geometries

Reviewer: ```suggestion
Delete detailed geometries
```

----------------------

> .. attention:: Removing features directly from **wastewater_structure** will remove these entities from the database and not only the detailled geometry!

Reviewer: ```suggestion
.. attention:: Deleting features directly from **wastewater_structure** will remove the complete wastewater structure from the database and not only the geometry of the wastewater structure!
```
